mmif/utils/textdoc_helper.py

```diff
@@ -0,0 +1,12 @@
+import mmif
+from mmif import Annotation
+
+
+def slice_text(mmif_obj, start: int, end: int, unit: str = "milliseconds") -> str:
+    token_type = "http://vocab.lappsgrid.org/Token"
+    anns_found = mmif_obj.get_annotations_between_time(start, end, unit)
+    tokens_sliced = []
+    for ann in anns_found:
+        if ann.is_type(token_type):
+            tokens_sliced.append(ann.get_property('text'))  # FIXME: Sometimes the string attribute "word" is used for getting property value
+    return ' '.join(tokens_sliced)
```

Comment:
Yeah, the `Token` type is actually a part of LAPPS vocabulary, and the property name should be `word` (https://vocab.lappsgrid.org/Token.html) So I can say that kaldi app has a ill-formed output. 

We have discussed merging two vocabularies, https://github.com/clamsproject/mmif/issues/202, but made no action so far. 

In any case, to use an enum class for lapps vocab items, we need to separately `pip install lapps`, but I don't see that's a good move. So having hardcoded raw string, I think, actually a good solution for now.